owner-site/site/docs/importing-properties.md

```diff
@@ -37,6 +37,16 @@ assertEquals("lime", cfg.bar());
 assertEquals("orange", cfg.baz());
 ```
 
+<div class="note info">
```

Comment:
Thanks, but it's better to avoid html in markdown. I'll merge and remove the html tags.

Reply:
My mistake :-), in this case is more than ok! please ignore above comment :)